tools/generate_opcodes_all.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+
+# This script generates internal/vm/opcodes/all.go.
+
+set -e
+
+temp_file=$(mktemp)
+
+(
+	# shellcheck disable=SC2016
+	echo '// Generated by generate_opcodes_all.sh. Run `make fix` to update. DO NOT EDIT.'
+	echo
+	echo 'package opcodes'
+	echo
+	echo 'import "github.com/git-town/git-town/v19/internal/vm/shared"'
+	echo
+	echo '// All provides all existing opcodes.'
+	echo '// This is used to iterate all opcode types.'
+	echo 'func All() []shared.Opcode {'
+	echo '	return []shared.Opcode{'
+	cat internal/vm/opcodes/*.go | grep '^type .* struct .*$' | sed -e 's/type /\t\t\&/' -e 's/ .*/{},/'
+	echo '	} //exhaustruct:ignore'
+	echo '}'
+) >"$temp_file"
+
+mv "$temp_file" internal/vm/opcodes/all.go
```

Comment:
Curious, why not update `internal/vm/opcodes/all.go` directly? I don't see any risks of it ending up in an inconsistent state, so wondering if there's a reason for going the detour through the temp file.

Reply:
I think I was trying to be extra cautious, but I agree that it's not necessary. I'll remove the detour.